packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.5
```

Comment:
Because there is no public API change, this should be 0.3.4+1

Reply:
Done